step-templates/slack-detailed-notification.json

```diff
@@ -3,7 +3,7 @@
n\t\t\t\tshort = \"true\";\r\n\t\t\t}\r\n\t\t;\t\r\n\t}\r\n\t\r\n\t\r\n\t$IncludeFieldReleaseNotes = [boolean]::Parse($OctopusParameters['IncludeFieldReleaseNotes'])\r\n\tif ($StatusInfo[\"success\"] -and $IncludeFieldReleaseNotes) {\r\n\t\r\n\t\t\r\n\t\t$link = $OctopusParameters['Octopus.Web.ReleaseLink'];\r\n\t\t$baseurl = $OctopusParameters['OctopusBaseUrl'];\r\n\t\t\r\n\t\t$notes = $OctopusReleaseNotes\r\n\t\t\r\n\t\tif ($notes.Length -gt 300) {\r\n\t\t\t$shortened = $OctopusReleaseNotes.Substring(0,0);\r\n\t\t\t$notes = \"$shortened `n `<${baseurl}${link}|view all changes`>\"\r\n\t\t}\r\n\t\t\r\n\t\t$fields +=  \r\n\t\t\t@{\r\n\t\t\t\ttitle = \"Changes in this release\";\r\n\t\t\t\tvalue = $notes;\r\n\t\t\t}\r\n\t\t;\t\r\n\t}\t\r\n\t\r\n\t#failure fields\r\n\t\r\n\t$IncludeErrorMessageOnFailure = [boolean]::Parse($OctopusParameters['IncludeErrorMessageOnFailure'])\r\n\tif (-not  $StatusInfo[\"success\"] -and $IncludeErrorMessageOnFailure) {\r\n\t\t\t\r\n\t\t$fields += \r\n\t\t\t@{\r\n\t\t\t\ttitle = \"Error text\";\r\n\t\t\t\tvalue = $OctopusParameters['Octopus.Deployment.Error'];\r\n\t\t\t}\r\n\t\t;\t\r\n\t}\t\r\n\t\t\r\n\r\n\t$IncludeLinkOnFailure = [boolean]::Parse($OctopusParameters['IncludeLinkOnFailure'])\r\n\tif (-not $StatusInfo[\"success\"] -and $IncludeLinkOnFailure) {\r\n\t\t\r\n\t\t$link = $OctopusParameters['Octopus.Web.DeploymentLink'];\r\n\t\t$baseurl = $OctopusParameters['OctopusBaseUrl'];\r\n\t\r\n\t\t$fields += \r\n\t\t\t@{\r\n\t\t\t\ttitle = \"See the process\";\r\n\t\t\t\tvalue = \"`<${baseurl}${link}|Open process page`>\";\r\n\t\t\t\tshort = \"true\";\r\n\t\t\t}\r\n\t\t;\t\r\n\t}\r\n\t\r\n\t\r\n\treturn $fields;\r\n\t\r\n}\r\n\r\nfunction Slack-Rich-Notification ($Success)\r\n{\r\n    $status_info = Slack-Populate-StatusInfo -Success $Success\r\n\t$fields = Slack-Populate-Fields -StatusInfo $status_info\r\n\r\n\t\r\n\t$payload = @{\r\n        channel = $OctopusParameters['Channel']\r\n        username = $OctopusParameters['Username'];\r\n        icon_url = $OctopusParameters['IconUrl'];\r\n\t\t\r\n        attachments = @(\r\n            @{\r\n\t\t\t\tfallback = $status_info[\"fallback\"];\r\n\t\t\t\tcolor = $status_info[\"color\"];\r\n\t\t\t\r\n\t\t\t\tfields = $fields\r\n            };\r\n        );\r\n    }\r\n\t\r\n\t#We unescape here to allow links in the Json, as ConvertTo-Json escapes <,> and other special symbols\r\n\t$json_body = ($payload | ConvertTo-Json -Depth 4 | % { [System.Text.RegularExpressions.Regex]::Unescape($_) });\r\n\t\r\n\t\r\n    try {\r\n        \r\n        Invoke-RestMethod -Method POST -Body $json_body  -Uri $OctopusParameters['HookUrl']  -ContentType 'application/json'    \r\n        \r\n    } catch {\r\n        echo \"Something occured\"\r\n        echo  $_.Exception\r\n        echo  $_\r\n        #echo $json_body\r\n        exit 0\r\n    }\r\n    \r\n}\r\n\r\n\r\n\r\n$success = ($OctopusParameters['Octopus.Deployment.Error'] -eq $null);\r\n\r\nSlack-Rich-Notification -Success $success\r\n",
```

Comment:
I think you might have forgotten to update the script to include the Tenant field in the resulting Slack message

Reply:
Oops, somehow I forgot to include this